toolchain/driver/driver_main.cpp

```diff
@@ -18,6 +19,16 @@ auto main(int argc, char** argv) -> int {
     return EXIT_FAILURE;
   }
 
+  // Find the executable without resolving symlinks. Do a PATH lookup if argv[0]
+  // isn't a valid path.
+  llvm::SmallString<128> exe_path(argv[0]);
```

Comment:
Guessing you want to use `exe_path` below and remove the call to `getMainExecutable`?

Reply:
Great idea :-)

Done.